src/Internal/QueryM/Ogmios.purs

```diff
@@ -654,7 +653,7 @@ type PoolParameters =
   , cost :: BigNum -- >= pparams.minPoolCost
   , margin :: UnitInterval -- proportion that goes to the reward account
   , rewardAccount :: RewardAddress
-  , poolOwners :: Array PaymentPubKeyHash
+  , poolOwners :: Array Ed25519KeyHash
```

Comment:
Why does that type change?

Reply:
Because I changed the instance, that is now not usable here.